docs/sources/tempo/metrics-generator/service_graphs/_index.md

```diff
@@ -71,10 +71,18 @@ The following metrics are exported:
 
 Duration is measured both from the client and the server sides.
 
-Possible values for `connection_type`: unset, `messaging_system`, or `database`.
+Possible values for `connection_type`: unset, `virtual_node`, `messaging_system`, or `database`.
 
-Additional labels can be included using the `dimensions` configuration option.
+Additional labels can be included using the `dimensions` configuration option, or the `enable_extra_uninstrumented_services_label` option.
 
 Since the service graph processor has to process both sides of an edge,
 it needs to process all spans of a trace to function properly.
 If spans of a trace are spread out over multiple instances, spans are not paired up reliably.
+
+#### Activating enable_extra_uninstrumented_services_label
+
+The following label, with the corresponding values, will be added if this feature is enabled:
```

Comment:
We try to use non-gerund (-ing verbs) in headings and avoid passive voice. What do you think about this suggestion?

```suggestion
#### Activate `enable_extra_uninstrumented_services_label`

Activating this feature adds the following label and corresponding values:
```

Reply:
Sounds good, thank you!